src/Reflectivity-VariableBreakpoint-Extensions/ClyBreakOnAccessCommand.class.st

```diff
@@ -0,0 +1,20 @@
+"
+Commands to install a breakpoint on the selected variable accesses from the vars view in Calypso
+"
+Class {
+	#name : #ClyBreakOnAccessCommand,
+	#superclass : #ClyBreakOnVariableCommand,
+	#category : #'Reflectivity-VariableBreakpoint-Extensions'
+}
+
+{ #category : #execution }
+ClyBreakOnAccessCommand >> breakOn: variable [
+	VariableBreakpoint
+		watchVariable: variable name
+		in: variable definingClass
```

Comment:
Here it can be simple 
```Smalltalk
variable breakOnAccess
```

Reply:
It would work, but it also needs an object-centric API and all the variations read/write/access. This needs some brainstorm - maybe it's simple in the end but I do not wish to do it now.

I suggest to leave it as is and work on this problem separately through another PR.

Reply:
The more I look at your suggestion, the more I find it better.
So I'll look at it but in the next weeks, it's not that quick (sometimes we call the API on a vaariable, sometimes on a class, sometimes on a node and a method...).
Again, I would prefer a PR to improve infrastructure so the PR can be integrated and we can take time to brainstorm without blocking the feature.